blocks/read-more/read-more.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable function-call-argument-newline, object-curly-newline, function-paren-newline */
+import { p, strong } from '../../scripts/dom-helpers.js';
+
+export default async function decorate(block) {
+  // get line count from block variant
+  const linesClass = block.className.split(' ').find((className) => className.startsWith('lines-'));
+  let lines = '4';
```

Comment:
This could be configured by the author.

Reply:
I suppose we could add a new row, but this is a full layout (no columns) so I just added it as class variant and the author can change it there.